packages/aws-cdk-lib/aws-route53/lib/geo-location.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Routing based on geographical location.
+ */
+export class GeoLocation {
+  /**
+   * Geolocation resource record based on continent code.
+   * @param continentCode Code of the continent. Valid codes are AF (Africa), AN (Antarctica), AS (Asia),
+   * EU (Europe), OC (Oceania), NA (North America), SA (South America).
+   * @returns Continent-based geolocation record
+   */
+  public static continent(continentCode: string) {
+    return new GeoLocation(continentCode, undefined, undefined);
+  }
+
+  /**
+   * Geolocation resource record based on country code.
+   * @param countryCode Two-letter, uppercase country code for the country.
+   * See ISO 3166-1-alpha-2 code on the *International Organization for Standardization* website
+   * @returns Country-based geolocation record
```

Comment:
Let's link to the specific docs (both from AWS and the ones AWS links to)?

```suggestion
   * @see https://docs.aws.amazon.com/Route53/latest/APIReference/API_GeoLocation.html#Route53-Type-GeoLocation-CountryCode
   * @see https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
   * @returns Country-based geolocation record
```